### ISSUE TITLE
Showing full filename

### ISSUE THREAD

**Issue by Hodelian** [DirectoryLister/DirectoryLister#1168]
Hi. I wand to use a directory browsing script, So I use Directory Lister 3.12.3 for that matter, but when my file's name are too long, it didn't show the file's name completely, there is any way of any change that I need to do to correct this?

Thank you in advance.

**Comment by jult** [comment]
Yes, full name option please, and directories on top! Back to nginx fancyindexing because of this..

**Comment by vmario89** [comment]
got the same issue - on a mobile phone it looks like this:
![grafik](https://github.com/user-attachments/assets/3cd30ab9-0a15-4fa5-a93e-e0c78ae5d167)
